config/locales/en.controllers.yml

```diff
@@ -44,10 +44,10 @@ en:
 
     facility_statements:
       errors_html: We experienced the following errors. Please try again.<br/>%{errors}
-      send_statements:
+      create_statements:
         order_error: Order %{order_detail_id} was either not found or cannot be statemented at this time.
         no_selection: No orders selected
-        success_html: Notifications sent successfully to:<br/>%{accounts}
+        success_html: Statements made successfully for:<br/>%{accounts}
```

Comment:
I wonder if we should have separate success messages here as well like we do for the instructions.